src/lib.rs

```diff
@@ -1019,40 +998,40 @@ pub fn scpmat(alpha: f64, a: &Sprs) -> Sprs {
     c.i = a.i.clone();
     c.x = a.x.iter().map(|x| x + alpha).collect();
 
-    return c;
+    c
 }
 
 /// Scalar times sparse matrix. C = alpha * A
 ///
 /// # Example:
+/// ```rust
+/// let a = [
+///     vec![8., 8., 6., 6., 2.],
+///     vec![4., 9., 7., 5., 9.],
+///     vec![2., 3., 8., 4., 1.],
+///     vec![4., 7., 6., 8., 9.],
+///     vec![9., 1., 8., 7., 1.],
+/// ];
+/// let mut a_sparse = rsparse::data::Sprs::new();
+/// a_sparse.from_vec(&a);
+///
+/// let r = [
+///     vec![16., 16., 12., 12., 4.],
+///     vec![8., 18., 14., 10., 18.],
+///     vec![4., 6., 16., 8., 2.],
+///     vec![8., 14., 12., 16., 18.],
+///     vec![18., 2., 16., 14., 2.],
+/// ];
+/// let mut r_sparse = rsparse::data::Sprs::new();
+/// r_sparse.from_vec(&r);
+///
+/// // Multiply a by 2
+/// assert_eq!(
+///     rsparse::scxmat(2., &a_sparse).to_dense(),
+///     r_sparse.to_dense()
+/// );
 /// ```
-/// fn main(){
-///    let a = vec![
-///        vec![8., 8., 6., 6., 2.],
-///        vec![4., 9., 7., 5., 9.],
-///        vec![2., 3., 8., 4., 1.],
-///        vec![4., 7., 6., 8., 9.],
-///        vec![9., 1., 8., 7., 1.],
-///     ];
-///     let mut a_sparse = rsparse::data::Sprs::new();
-///     a_sparse.from_vec(&a);
-///
-///     let r = vec![
-///         vec![16., 16., 12., 12., 4.],
-///         vec![8., 18., 14., 10., 18.],
-///         vec![4., 6., 16., 8., 2.],
-///         vec![8., 14., 12., 16., 18.],
-///         vec![18., 2., 16., 14., 2.],
-///     ];
-///     let mut r_sparse = rsparse::data::Sprs::new();
-///     r_sparse.from_vec(&r);
-///
-///     // Multiply a by 2
-///     assert_eq!(
-///         rsparse::scxmat(2., &a_sparse).to_dense(),
-///         r_sparse.to_dense()
-///     );
-/// }
+
 /// ```
```

Comment:
Minor fix: This code block closes twice

Reply:
Thanks for pointing this out!
Fixed by [this commit](https://github.com/RLado/rsparse/pull/7/commits/71d0e41cecd7f6992ca06a1bf886465002487256)

Tests still pass ✅